roamance/resource/roamance-srs-client.md

```
@@ -0,0 +1,160 @@
+# Roamance Frontend Requirements Specification
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)